run_dir/design/worksets.html

```diff
@@ -0,0 +1,90 @@
+{% extends "base.html" %}
+
+<!-- 
+Template file: worksets.html
+URL: /worksets/[optional filter]
+Title: Summary of all Worksets
+Description: Shows a table with all worksets.
+-->
+
+{% block stuff %}
+
+{% if worksets == 'all' %}
+  <h1 id="{{ worksets }}" class="worksets_page_heading"><span id="page_title">All Worksets</span> <small>All worksets found in statusdb</small></h1>
+{% end %}
+
+
+<div id="loading_spinner" style="text-align:center; margin:100px 0;"><span class='glyphicon glyphicon-refresh glyphicon-spin'></span>  Loading worksets table..</div>
+<div id="page_content" style="display:none;">
+
+<form id="Search-form" class="form-inline marginBottom">
+  <div class="pull-right">
```

Comment:
Ok.. I'd be inclined to delete it all and then copy it back in from the Flowcells page if needed, to keep things clean. But it's up to you - you'll probably be the one extending it later, so if you'd rather comment it out then that's fine.